openquake/server/urls.py

```diff
@@ -23,24 +23,27 @@
 from openquake.server import views
 
 urlpatterns = [
-    url(r'^$', RedirectView.as_view(url='/engine/', permanent=True)),
     url(r'^v1/engine_version$', views.get_engine_version),
     url(r'^v1/engine_latest_version$', views.get_engine_latest_version),
     url(r'^v1/calc/', include('openquake.server.v1.calc_urls')),
     url(r'^v1/valid/', views.validate_nrml),
     url(r'^v1/available_gsims$', views.get_available_gsims),
     url(r'^v1/on_same_fs$', views.on_same_fs, name="on_same_fs"),
-    url(r'^engine/?$', views.web_engine, name="index"),
-    url(r'^engine/(\d+)/outputs$',
-        views.web_engine_get_outputs, name="outputs"),
-    url(r'^engine/license$', views.license,
-        name="license"),
 ]
 
-for app in settings.STANDALONE_APPS:
-    app_name = app.split('_')[1]
-    urlpatterns.append(url(r'^%s/' % app_name, include('%s.urls' % app,
-                           namespace='%s' % app_name)))
+if settings.WEBUI:
+    urlpatterns += [
```

Comment:
Please add a comment that it is useful to disable the redirect if the usage if via API only